databuilder/tests/unit/task/test_neo4j_staleness_removal_task.py

```diff
@@ -222,26 +271,26 @@ def test_delete_statement_publish_tag(self) -> None:
             mock_execute.assert_any_call(dry_run=False,
                                          param_dict={'marker': u'foo', 'batch_size': 100},
                                          statement=textwrap.dedent("""
-            MATCH (n:Foo)
+            MATCH (target:Foo)
```

Comment:
I'm seeing a lot of repetition of the same template (mostly by previous authors). Could you pull out all/most of the query templates into class variables in the code and then just reference them both in the code and in the tests so you don't have to type out this boilerplate each time.